src/plugins/console/public/application/components/import_flyout.tsx

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiCallOut,
+  EuiSpacer,
+  EuiFilePicker,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiLoadingSpinner,
+  EuiText,
+  EuiButton,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import moment from 'moment';
+import { i18n } from '@osd/i18n';
+import { FormattedMessage } from '@osd/i18n/react';
+import React, { Fragment, useState } from 'react';
+import { ImportMode, ImportModeControl } from './import_mode_control';
+import { useEditorReadContext, useServicesContext } from '../contexts';
+import { TextObject } from '../../../common/text_object';
+import { OverwriteModal } from './overwrite_modal';
+
+const OVERWRITE_ALL_DEFAULT = false;
+
+interface ImportFlyoutProps {
+  close: () => void;
+  refresh: () => void;
+}
+
+const getErrorMessage = () => {
+  return i18n.translate('console.ImportFlyout.importFileErrorMessage', {
+    defaultMessage: 'The file could not be processed due to error.',
+  });
+};
```

Comment:
I have some concerns about the static error message here. It doesn't give enough info to user to trouble shoot. Can we dynamically configure the error message from the actual error?